File: VB/WpfApplication1/WizardCustomization/CustomReportWizardLauncherService.vb
#Region "#ReportWizardLauncherService"
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Data.WizardFramework
Imports DevExpress.Data.XtraReports.ServiceModel
Imports DevExpress.Data.XtraReports.Wizard
Imports DevExpress.Xpf.ReportDesigner.Services
' ...

Namespace WpfApplication1.WizardCustomization
	Friend Class CustomReportWizardLauncherService
		Inherits ReportWizardLauncherService(Of ReportModel)
        Protected Overrides Function CreatePageFactory(ByVal client As IReportWizardServiceClient) As IWizardPageFactory(Of ReportModel)
            Return New CustomReportDesignerWizardPageFactory(client)
        End Function
	End Class
End Namespace
#End Region ' #ReportWizardLauncherService

File: CS/WpfApplication1/WizardCustomization/CustomReportWizardLauncherService.cs
#region #ReportWizardLauncherService
using DevExpress.Data.WizardFramework;
using DevExpress.Data.XtraReports.ServiceModel;
using DevExpress.Data.XtraReports.Wizard;
using DevExpress.Xpf.ReportDesigner.Services;
// ...

namespace WpfApplication1.WizardCustomization {
    class CustomReportWizardLauncherService : ReportWizardLauncherService<ReportModel> {
        protected override IWizardPageFactory<ReportModel> CreatePageFactory(IReportWizardServiceClient client) {
            return new CustomReportDesignerWizardPageFactory(client);
        }
    }
}
#endregion #ReportWizardLauncherService